src/Chapter1.hs

```diff
@@ -631,8 +634,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2::Int -> Int
+sumLast2 n = last n + prelast n
+  where
+    last::Int -> Int
+    last n = mod n 10
+    prelast::Int -> Int
+    prelast n = mod (div n 10) 10
```

Comment:
You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both in `last` and `prelast`:

```haskell
mod smth 10
div smth 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂